ush/ioda/bufr2ioda/bufr2ioda_adpsfc_prepbufr.py

```diff
@@ -0,0 +1,341 @@
+    logger.info(f" ... ... Create variables: name, type, units, & attributes")
+
+    # Observation Type - Station Elevation
+    obsspace.create_var('ObsType/stationElevation', dtype=typ.dtype,
+                        fillval=typ.fill_value) \
+        .write_attr('long_name', 'Station Elevation Observation Type') \
+        .write_data(typ)
+
+    # Observation Type - Station Pressure
+    obsspace.create_var('ObsType/stationPressure', dtype=typ.dtype,
+                        fillval=typ.fill_value) \
+        .write_attr('long_name', 'Station Pressure Observation Type') \
+        .write_data(typ)
+
+    # Longitude
+    obsspace.create_var('MetaData/longitude', dtype=lon.dtype,
+                        fillval=lon.fill_value) \
+        .write_attr('units', 'degrees_east') \
+        .write_attr('valid_range', np.array([-180, 180], dtype=np.float32)) \
+        .write_attr('long_name', 'Longitude') \
+        .write_data(lon)
+
+    # Latitude
+    obsspace.create_var('MetaData/latitude', dtype=lat.dtype,
+                        fillval=lat.fill_value) \
+        .write_attr('units', 'degrees_north') \
+        .write_attr('valid_range', np.array([-90, 90], dtype=np.float32)) \
+        .write_attr('long_name', 'Latitude') \
+        .write_data(lat)
+
+    # Datetime
+    obsspace.create_var('MetaData/dateTime', dtype=dateTime.dtype,
+                        fillval=dateTime.fill_value) \
+        .write_attr('units', 'seconds since 1970-01-01T00:00:00Z') \
+        .write_attr('long_name', 'Datetime') \
+        .write_data(dateTime)
+
+    # Station Identification
+    obsspace.create_var('MetaData/stationIdentification', dtype=sid.dtype,
+                        fillval=sid.fill_value) \
+        .write_attr('long_name', 'Station Identification') \
+        .write_data(sid)
+
+    # Height Of Station
+    obsspace.create_var('MetaData/heightOfStation', dtype=zob.dtype,
+                        fillval=zob.fill_value) \
+        .write_attr('units', 'm') \
+        .write_attr('long_name', 'Height Of Station') \
+        .write_data(zob)
+
+    # Pressure
+    obsspace.create_var('MetaData/pressure', dtype=pressure.dtype,
+                        fillval=pressure.fill_value) \
+        .write_attr('units', 'Pa') \
+        .write_attr('long_name', 'Pressure') \
+        .write_data(pressure)
+
+    # QualityMarker - Station Elevation
+    obsspace.create_var('QualityMarker/stationElevation', dtype=zobqm.dtype,
+                        fillval=zobqm.fill_value) \
+        .write_attr('long_name', 'Station Elevation Quality Marker') \
+        .write_data(zobqm)
+
+    # QualityMarker - Station Pressure
+    obsspace.create_var('QualityMarker/stationPressure', dtype=pobqm.dtype,
+                        fillval=pobqm.fill_value) \
+        .write_attr('long_name', 'Station Pressure Quality Marker') \
+        .write_data(pobqm)
+
+    # Station Elevation
+    obsspace.create_var('ObsValue/stationElevation', dtype=elv.dtype,
```

Comment:
is this an `ObsValue`?

Reply:
stationElevation (mnemonic elv) is an ObsValue while heightOfStation (mnemonic zob) is a MetaData